_posts/2020-12-04-windows.md

```diff
@@ -0,0 +1,578 @@
+---
+layout: post
+title: Trigger to generate Windows boot files
+author: Yuriy
+summary: Automatic network Windows installation without Microsoft tools
+---
+
+## Cobbler Windows installation
+
+Many companies now have a separate infrastructure for deploying Linux servers and Windows workstations, which leads to additional costs. This kind of overhead can be reduced by using a single platform to install both kinds of operating systems.
+
+Despite the success of Microsoft in creating tools for integrating with Linux, in my opinion, Linux is more suitable for such a platform thanks to the wonderful project Cobbler. Cobbler is incredibly flexible and can be tailored to suit any need. Using templates and Cobbler profiles with the ability to inherit (parent profile), you can change a couple of lines in the template, you can very quickly create an installation option adapted to the needs of a particular user without having to depend on the capabilities of the closed proprietary code.
+
+Cobbler has an excellent architectural feature of expanding functionality by writing triggers that do not affect the main code. We used this opportunity to prepare the necessary files for a Windows installation.
+
+Linux currently has all the necessary tools to prepare these files. All that remains is to combine these tools in the trigger code to get an open source infrastructure for deploying Windows without using proprietary Microsoft tools.
+
+Trigger sync_post_wingen.py does:
+- some of the files are created from standard ones (`pxeboot.n12`, `bootmgr.exe`) by directly replacing one sting with another directly in the binary
```

Comment:
string instead of sting

Reply:
@dansou901 I haven't studied git very well yet, and maybe my commit https://github.com/tpw56j/cobbler.github.io/commit/42552f625495ed06896763226e474d9e462c914f was not included in the PR?